addon/src/-private/class-based/modifier-manager.ts

```diff
@@ -57,7 +58,7 @@ export default class ClassBasedModifierManager<S> {
     dependencySatisfies('ember-source', '^3.22.0 || ^4.0.0') ? '3.22' : '3.13'
   );
 
-  constructor(private owner: unknown) {}
+  constructor(private owner: Owner) {}
```

Comment:
@chriskrycho do I get it right we should change the signature to adhere https://rfcs.emberjs.com/id/0821-public-types/#owner ?

as we are in v4.beta, this might be a good timing to do so as it's breaking change per types policy.
unless I'm missing something

Reply:
Yep, this is the right thing to do (and sorry for the long-delayed response)!